metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1689,7 +1692,7 @@ def _convert_to_source(self, result: Mapping[str, Any]) -> Source:
 
     def _convert_to_statistics(self, result: Mapping[str, Any]) -> Stat:
         return Stat(
-            stat_type=_safe_get(result, 'stat_type'),
+            stat_type=_safe_get(result, 'stat_name'),
```

Comment:
From: https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/models/table_stats.py#L87

Reply:
Again a mismatch between amundsen common and databuilder: https://github.com/amundsen-io/amundsencommon/blob/master/amundsen_common/models/table.py#L64

Reply:
this seems to be an issue for amundsendatabuilder. cc @allisonsuarez

Reply:
Let me know how you would to handle this @allisonsuarez. I don't mind making the changes in both databuilder and metadata.

Reply:
let's change the databuilder to make it consistent.